doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -57,107 +57,95 @@ Values are provided in kilobytes (KB).
-      | :ref:`Thermostat <matter_thermostat_sample>` (Debug)             |            32 |               695 |              4 |         32 |         763 |                             167 |
+      | :ref:`Thermostat <matter_thermostat_sample>` (Release)           |            32 |               563 |              4 |         32 |         631 |                             165 |
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Thermostat <matter_thermostat_sample>` (Release)           |            32 |               592 |              4 |         32 |         660 |                             160 |
+      | :ref:`Window Covering <matter_window_covering_sample>` (Debug)   |            32 |               657 |              4 |         32 |         725 |                             172 |
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Window Covering <matter_window_covering_sample>` (Debug)   |            32 |               695 |              4 |         32 |         763 |                             167 |
-      +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Window Covering <matter_window_covering_sample>` (Release) |            32 |               595 |              4 |         32 |         663 |                             160 |
+      | :ref:`Window Covering <matter_window_covering_sample>` (Release) |            32 |               555 |              4 |         32 |         623 |                             165 |
       +------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
 
    .. tab:: Thingy:53
 
       The following table lists memory requirements for samples running on the :ref:`Thingy:53 <programming_board_names>` (:ref:`thingy53_nrf5340 <zephyr:thingy53_nrf5340>`).
 
       +---------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | Sample                                                        |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      | Sample                                                        |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```

Comment:
```suggestion
      | Sample                                                        |   MCUboot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
```

Reply:
Hmm, So I will update the script for generating the data also :)